bundles/org.openhab.binding.homematic/README.md

```diff
@@ -478,6 +478,115 @@ then
     Display_submit.sendCommand(ON)
 end
 ```
+
+### HmIP-WRCD
+
+The HmIP-WRCD display lines can be set via a combined parameter:
+
+```java
+String Display_CombinedParam "Combined Parameter" {channel="homematic:HmIP-WRCD:ccu:123456:3#COMBINED_PARAMETER"}
+```
+#### Set Display Lines
+
+The combined parameter can be used in the console:
+```shell
+smarthome:send Display_CombinedParam '{DDBC=WHITE,DDTC=BLACK,DDI=0,DDA=CENTER,DDS=Just a test,DDID=3,DDC=true}'
+```
```

Comment:
It would be better to have rule coding as an example.

Reply:
Okay, added a rule coding example. I've still kept the shell example, because it cost me several hours until I've figured out that the curly braces get evaluated when using " instead of '.